src/xlsx/utils.nim

```diff
@@ -10,8 +10,7 @@ const
   UpperLetters = {'A' .. 'Z'}
   CharDataOption = {xmlCharData, xmlWhitespace}
   DEFAULT_WORKBOOK_PATH = "xl/workbook.xml"
-let TempDir* = getTempDir() / "xlsx_windx_tmp" ## temp dir for all extracted xml files from Excel
-
+  XLSX_TMP = "xlsx_windx_tmp"
```

Comment:
You can use `const` instead, since `getTempDir` works at compile time.

Reply:
Yes, that's better. I had just assumed it couldn't be since it wasn't already.